_docs/configuration.md

```diff
@@ -237,4 +237,15 @@ The ruleset is built and applied as follows:
   .deny("*.acme.com")
   .build()
 )
+```
+
+## Filename template
+
+WireMock can set up specific filename template format based on stub information. 
```

Comment:
Would it make sense to specify a version here? WireMock 3.0 is nowhere here, and it may confuse users


```suggestion
Starting from [3.0.0-beta-8](https://github.com/wiremock/wiremock/releases/tag/3.0.0-beta-8),
WireMock can set up specific filename template format based on stub information. 
```